packages/react-native-passkey-stamper/src/util.ts

```diff
@@ -0,0 +1,45 @@
+// Buffer is available in Node and react-native contexts but needs to be imported
+import { Buffer } from "buffer";
+import { createHash } from "sha256-uint8array";
+
+// Needs to return a base64-encoded string
+export function getChallengeFromPayload(payload: string): string {
+  const hexString = createHash().update(payload).digest("hex");
+  const hexBuffer = Buffer.from(hexString, "utf8");
+  return hexBuffer.toString("base64");
+}
+
+// Function to return 32 random bytes encoded as hex
+// (e.g "5e4c2c235fc876a9bef433506cf596f2f7db19a959e3e30c5a2d965ec149d40f")
+// This function doesn't return strong cryptographic randomness (Math.random is a PRNG), but this is good enough for registration challenges.
+// If the challenge was not random at all the risk is that someone can replay a previous signature to register an authenticator they don't own.
+// However:
+// - we are creating a brand new authenticator here, which means keygen is happening right as we call this library (which makes the replay attack hard-to-impossible)
+// - even if a replay attack went through, the authenticator wouldn't be usable given Turnkey has anti-replay in place in activity payloads (timestamp field)
```

Comment:
optional: can link to the small blurb we have! 
https://docs.turnkey.com/api-introduction#queries-and-submissions

though on the flip side, there's no guarantee this link will live forever as-is.

Reply:
Let's do it 👍